pkg/kube/controllers/boshdeployment/validating_webhook.go

```diff
@@ -34,7 +34,7 @@ func NewBOSHDeploymentValidator(log *zap.SugaredLogger, config *config.Config) *
 		Rules: []admissionregistrationv1beta1.RuleWithOperations{
 			{
 				Rule: admissionregistrationv1beta1.Rule{
-					APIGroups:   []string{"fissile.cloudfoundry.org"},
+					APIGroups:   []string{"quarks.cloudfoundry.org"},
```

Comment:
any reason why you are not using the https://github.com/cloudfoundry-incubator/quarks-utils/blob/af677c684e378cce33c908f8ef640ed39b60e78f/pkg/names/apigroup.go#L4 in here?